tea/scenes/profile.go

```diff
@@ -22,7 +22,7 @@ type profile struct {
 	hadRenamed bool
 }
 
-func NewProfile(root components.RootModel, parent tea.Model, profileData *cli.Profile) tea.Model {
+func EditProfile(root components.RootModel, parent tea.Model, profileData *cli.Profile) tea.Model {
```

Comment:
Do not rename this, please follow the Go convention of calling struct constructors `New<struct>`